python/pyspark/sql/pandas/conversion.py

```diff
@@ -117,6 +118,9 @@ def toPandas(self):
                             if isinstance(field.dataType, TimestampType):
                                 pdf[field.name] = \
                                     _check_series_localize_timestamps(pdf[field.name], timezone)
+                            elif isinstance(field.dataType, MapType):
+                                pdf[field.name] = \
+                                    _convert_map_items_to_dict(pdf[field.name])
```

Comment:
Quick question @BryanCutler. So PyArrow converts Arrow's map type to a list of tuples by default?